### ISSUE TITLE
InAppBrowser disables touch or lost focus after keyboard shows/hides

### ISSUE THREAD

**Issue by andaralex** [apache/cordova-plugin-inappbrowser#664]
I have loaded a chat window URL (web application) in a mobile app using the inAppBrowser. The web page is loaded correctly and works fine in IAB. But while I start typing the message in the chat window keypad open and web content moves up and now I hide the keypad, the page will not come to normal position as well the page lost their focus and disabled the touch and all. 

Once I pinch/zoom in -out, the page it comes to normal position then I can able to start typing and click the icons.

**Anyone has an idea to resolve this issue?**

The same page works fine in mobile browser Safari as well as Ionic SafariViewController.

**Comment by codehack26** [comment]
> I have loaded a chat window URL (web application) in a mobile app using the inAppBrowser. The web page is loaded correctly and works fine in IAB. But while I start typing the message in the chat window keypad open and web content moves up and now I hide the keypad, the page will not come to normal position as well the page lost their focus and disabled the touch and all.
> 
> Once I pinch/zoom in -out, the page it comes to normal position then I can able to start typing and click the icons.
> 
> **Anyone has an idea to resolve this issue?**
> 
> The same page works fine in mobile browser Safari as well as Ionic SafariViewController.

Yes I getting the same issue did you find any solution?

**Comment by andaralex** [comment]
@codehack26 
Yes, we have fixed this issue using cordova-plugin-wkkeyboardfix.

here - Fix(iOS): Issues related to keyboard dismissal (4.x) ionic-team/cordova-plugin-ionic-webview#533, they recommended to use cordova-plugin-wkkeyboardfix.

Just for your reference, these are the plugins I have used in my app.
 "cordova-plugin-inappbrowser": "3.2.0",
 "cordova-plugin-ionic-keyboard": "2.2.0",
 "cordova-plugin-ionic-webview": "4.2.1",
  "cordova-plugin-wkkeyboardfix": "1.1.0",
